### DISCUSSION TITLE
User login and auth

### DISCUSSION THREAD

**Discussion by JokerMartini** [quasarframework/quasar#10487] [General - Components / Directives / etc]
First off I'm a huge fan of this framework. I'm looking to develop an application that contains user roles and user login authorization. I was wondering if anyone has recommendations on Frameworks to use with Quasar or plug-ins to achieve this?

I know vuetify apps using nuxt use https://auth.nuxtjs.org/ for this stuff. Looking to do this with quasar. 

Thank you!

**Comment by MilosPaunovic** [comment]
Have you tried this, from the first page of Google results:
https://github.com/stefanvanherwijnen/quasar-app-extension-http-authentication

**Reply by JokerMartini** [reply]
Sadly it does not support vue3
